<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewConfigDialog</class>
 <widget class="QDialog" name="ViewConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VstBoard Appearance</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="0" colspan="5">
           <widget class="QCheckBox" name="checkSavedInSetupFile">
            <property name="toolTip">
             <string>Customize colors of the current setup file</string>
            </property>
            <property name="text">
             <string>Saved in setup file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QListWidget" name="listPresets">
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Presets :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="addPreset">
            <property name="toolTip">
             <string>Add new preset</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/img16x16/edit_add.png</normaloff>:/img16x16/edit_add.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ins</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QToolButton" name="delPreset">
            <property name="toolTip">
             <string>Remove preset</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/img16x16/edit_remove.png</normaloff>:/img16x16/edit_remove.png</iconset>
            </property>
            <property name="shortcut">
             <string>Del</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Objects :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Elements :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="listPalettes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListWidget" name="listRoles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_3">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupColor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Color</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Red</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="RedSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="RedSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="GreenSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSpinBox" name="GreenSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Green</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSlider" name="BlueSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Blue</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSpinBox" name="BlueSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSlider" name="AlphaSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Alpha</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QSpinBox" name="AlphaSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="3">
              <widget class="View::GradientWidget" name="picker" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="3">
              <widget class="View::GradientWidgetHue" name="picker_hue" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Programs list :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="fontProgSize">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QCheckBox" name="fontProgItalic">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Family</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fontProgFamily"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Stretch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QDoubleSpinBox" name="fontProgStretch">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="fontProgBold">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View::GradientWidget</class>
   <extends>QWidget</extends>
   <header>views/gradientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>View::GradientWidgetHue</class>
   <extends>QWidget</extends>
   <header>views/gradientwidgethue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>listPresets</tabstop>
  <tabstop>checkSavedInSetupFile</tabstop>
  <tabstop>addPreset</tabstop>
  <tabstop>delPreset</tabstop>
  <tabstop>listPalettes</tabstop>
  <tabstop>listRoles</tabstop>
  <tabstop>RedSlider</tabstop>
  <tabstop>RedSpinBox</tabstop>
  <tabstop>GreenSlider</tabstop>
  <tabstop>GreenSpinBox</tabstop>
  <tabstop>BlueSlider</tabstop>
  <tabstop>BlueSpinBox</tabstop>
  <tabstop>AlphaSlider</tabstop>
  <tabstop>AlphaSpinBox</tabstop>
  <tabstop>fontProgFamily</tabstop>
  <tabstop>fontProgSize</tabstop>
  <tabstop>fontProgBold</tabstop>
  <tabstop>fontProgItalic</tabstop>
  <tabstop>fontProgStretch</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ViewConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
